VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift

```diff
@@ -115,14 +116,17 @@ struct KeygenView: View {
     }
     
     var appVersion: some View {
-        Text("Vultisig APP V\(version ?? "1")")
-            .textCase(.uppercase)
-            .font(.body14Menlo)
-            .foregroundColor(.turquoise600)
+        return VStack {
```

Comment:
can we please add this to keysign as well

Reply:
Created a separate issue for that since it'll require shuffling a bit of stuff around from UI perspective.
https://github.com/vultisig/vultisig-ios/issues/1126